app/lib/actions/katello/repository/destroy.rb

```diff
@@ -23,28 +24,37 @@ def plan(repository, options = {})
           plan_action(ContentViewPuppetModule::Destroy, repository) if repository.puppet?
           plan_action(Pulp::Repository::Destroy, pulp_id: repository.pulp_id)
           sequence do
-            plan_action(Product::ContentDestroy, repository)
-            plan_action(Repository::DestroyMedium, repository)
-
-            view_env = repository.content_view.content_view_environment(repository.environment)
             repository.destroy! if planned_destroy
-
-            if !skip_environment_update && ::SETTINGS[:katello][:use_cp] && view_env
```

Comment:
nah, not really.  It was used in the past for various reasons, but not really anymore